po/common/common.pot

```diff
@@ -4688,10 +4688,6 @@ msgctxt "query_filter"
 msgid "Query filter"
 msgstr "Query filter"
 
-msgctxt "nova_group_producer"
```

Comment:
@stephanegigandet you are deleting this !

Reply:
good catch, I have too many open branches